src/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,84 @@
+import { useContext, useMemo, useCallback } from 'react';
+
+import { TodosContext } from '../TodosContext/TodosContext';
+import { Status } from '../enum/Status';
+
+export const TodoFooter = () => {
+  const {
+    todos,
+    setTodos,
+    filterStatus,
+    setFilterStatus,
+  } = useContext(TodosContext);
+
+  const uncompletedTodos = useMemo(() => {
+    return todos.filter(todo => !todo.completed).length;
+  }, [todos]);
+
+  const clearCompletedTodo = useCallback(() => {
+    const todosToComplete = todos
+      .filter(todo => !todo.completed);
+
+    setTodos(todosToComplete);
+  }, [todos]);
+
+  return (
+    <>
+      {todos.length > 0 && (
+        <footer className="footer">
+          <span className="todo-count" data-cy="todosCounter">
+            {`${uncompletedTodos} items left`}
+          </span>
+
+          <ul className="filters" data-cy="todosFilter">
+            <li>
+              <a
+                href="#/"
+                className={
+                  Status.ALL === filterStatus ? 'selected' : ''
+                }
```

Comment:
let's use the classNames package for such cases

Reply:
agree, please use class name library

Reply:
still not fixed
<img width="651" alt="image" src="https://github.com/mate-academy/react_todo-app/assets/70220667/159ed5b6-c1ca-430d-ad00-a70acc903dda">